x-pack/plugins/ml/public/file_datavisualizer/components/import_summary/import_summary.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React from 'react';
+
+import {
+  EuiSpacer,
+  EuiDescriptionList,
+  EuiCallOut,
+  EuiAccordion,
+} from '@elastic/eui';
+
+export function ImportSummary({
+  index,
+  indexPattern,
+  ingestPipelineId,
+  docCount,
+  importFailures,
+}) {
+  const items = createDisplayItems(
+    index,
+    indexPattern,
+    ingestPipelineId,
+    docCount,
+    importFailures
+  );
+
+  return (
+    <React.Fragment>
+      <EuiCallOut
+        title="Import complete"
+        color="success"
+        iconType="check"
+      >
+        <div className="import-summary-list">
```

Comment:
As above, is the `div` necessary here, or could you just use a `className` prop on the `EuiDescriptionList`?